### PR TITLE
Fix nasted path and allow wildcard

### PR DESCRIPTION
**Pull request by pmoranga**

### CODE REVIEW

File: check_json.pl [hunk 1]
```diff
@@ -234,26 +277,57 @@
...
+        ## Handle case of wildcard in the middle of the tree: {data}->*->{description}
+        $output_value = join(", ",process_wildcard($key,$json_response));
         push(@statusmsg, "$label: $output_value");
     }
 }
 
 $np->nagios_exit(
     return_code => $result,
-    message     => join(', ', @statusmsg),
+    message     => join('; ', @statusmsg),
```
Comment (senorsmile) [inline check_json.pl:332]: what's the reason for changing this to a semi-colon?

Comment (pmoranga) [reply check_json.pl:332]: better readability.

Comment (senorsmile) [reply check_json.pl:332]: I believe nagios expects this to be separated by commas.  I fear it won't work correctly if that is changed.

Comment (pmoranga) [reply check_json.pl:332]: This is not perfdata, so there is no requirements as far I double checked in the docs. Let me know if you prefer me to revert it.
https://nagios-plugins.org/doc/guidelines.html#PLUGOUTPUT

Comment (senorsmile) [reply check_json.pl:332]: Ah you're right.  Although I think Icinga2 actually relies on it for formatting in the status.  If this breaks any of our users we can revert it.  

👍

**Pull request merged**